streaming/kafka/install.sh

```diff
@@ -12,11 +12,15 @@ targets=(
   "module.eks_data_addons"
 )
 
+# Initialize Terraform
+echo "Initializing Terraform..."
+terraform init
+
 # Apply modules in sequence
 for target in "${targets[@]}"
 do
   echo "Applying module $target..."
-  apply_output=$(terraform apply -target="$target" -auto-approve 2>&1 | tee /dev/tty)
+  apply_output=$(terraform apply -target="$target" -var=region=$(aws configure get region) -auto-approve 2>&1 | tee /dev/tty)
```

Comment:
This could potentially create issues for users who intend to deploy the cluster in a specific region as specified in the variables file.

Our `install.sh` script depends on the `region` variables defined in the blueprint. Users must take care to update the region within the variables file to match their intended deployment region. There are situations where their local region might be configured as `eu-west-1`, but their deployment target is `us-west-2` or `us-east-1`.

Reply:
this is also not "sound" behavior for a declarative IaC tool/framework - you declare the value in code, and apply that. using a dynamic lookup could lead to destroying an entire regions worth of resources